frontend/src/components/navbar.svelte

```diff
@@ -73,7 +96,7 @@
     >
       <option disabled={true}>Choose country</option>
       {#each countries as country}
-        <option value={country.value}>{country.name}</option>
+        <option value={country.value}>{country.icon} {country.name}</option>
       {/each}
     </select>
```

Comment:
The dropdown needs to be searchable now IMO - Could use the multiselect for this, I'm pretty sure it was a "don't be a multi-select"-option

Reply:
Hmmm, not sure if i agree - this would hurt mobile use quite a bit imo

Reply:
Fair point, we should probably not over-engineer a country selector :D